### ISSUE TITLE
No Project Zulu spawns in Terrain Control generated world

### ISSUE THREAD

**Issue by clubpetey** [soultek101/projectzulu1.7.10#23]
Cross posted on TerrainControl gitHub, but looking for some help here as well.

I've created a world with TC that is all Desert and Ocean. In the ocean I've set the water level to 10, and changed the blocks to generate Sand/Sandstone.

I have a bunch of mods installed but the one that is giving me trouble is Project Zulu. I've found a couple of weird things occuring, so I will post this in both githubs in hopes some can work it out.
1. PZ mobs do not spawn, I can't find any of them anywhere. Standard mobs do spawn, but not in the right places. (See #3 and #4)
2. PZ terrain features (pyramid, etc) do not spawn. I wonder if TC changes the class/name of the biomes from the MC standard class, since that seems to be what PZ looks form.
3. Non-hostile mobs (Chicken, Cows, etc) spawn in the ocean. According to MC wiki, this isn't supposed to happen. Not sure if TC or PZ is overriding the default spawn rules.
4. Witches seem to be spawning anywhere on the surface.

Thanks for your help!

MC Version: 1.7.10
TC Version: 2.6.3
PZ Version: 1.4c2

**Comment by clubpetey** [comment]
Update: after talking with TC guys...

"All biomes use com.khorn.terraincontrol.forge.generator.BiomeGenCustom"

So I added lines like this to the ProjectZuluMobBiomeConfig.cfg:

sandworm {
        S:com.khorn.terraincontrol.forge.generator.BiomeGenCustom.Desert=true:1:1:1
        S:com.khorn.terraincontrol.forge.generator.BiomeGenCustom.Ocean=true:1:1:1
...
}

Still no spawns tho.  Did I mis-read the format?  Looks like it's "class_name.biome_name"

**Comment by soultek101** [comment]
Hi there. So sorry for my late response as RL has a good chocke hold on my time. I'll see what I can do but it has been reported to the PZ thread before that PZ mobs and structures do not spawn on a pre generated world. I was able to use an old mod that allows for forced respawning of mobs/blocks and such and was able to test it to a degree. I'll look up my old archives and see the name and hopefully that can help out.

**Comment by clubpetey** [comment]
I don't think this is an issue with pre-generated worlds.  I've started over with new worlds, and went into new chunks of existing worlds. No PZ spawns.
